Mono.Debugging/Mono.Debugging.Client/BreakpointStore.cs

```diff
@@ -73,21 +77,17 @@ static bool IsRunningOnMac ()
 			}
 			return false;
 		}
-
-		readonly List<BreakEvent> breakpoints = new List<BreakEvent> ();
 		
 		public int Count {
 			get {
-				return breakpoints.Count;
+				return InternalGetBreakpoints().Length;
 			}
```

Comment:
Not sure we need to lock to check the length of an ImmutableArray.

Reply:
No, not necessary.

Reply:
Only for the reason below about atomicity.